Home.md

```diff
@@ -1,40 +1,51 @@
+
+### d:swarm *DataHub* Version
+
+Archiving versions of the transformed data is only possible with the *DataHub* version of d:swarm, which is also the basis for [[upcoming functionality | Outlook]] such as deduplication, frbrization and other data quality improvements. While many steps into this direction have been taken, challenges remain with respect to scalability for very large datasets. See [this blog post](http://www.slub-dresden.de/blog-post-on-challenges-property-graph-handling).
+
+<!--- source code for image currently stored by jan -->
+[[img/dswarm-usage-variants.png]]
 
 ### ... and behind the scenes
 
-As shown below, the overall architecture consists of two major parts: the Backoffice [[front end]] and the [[back end]]. The back end consists of 4 modules: 
+As shown below, the overall architecture consists of three major parts: the [BackOffice web application](http://demo.dswarm.org), the [TPU](https://github.com/dswarm/task-processing-unit-for-dswarm), and the [[back end]]. The back end, in turn, consists of three modules: 
  * a [[controller]] module that controls the program flow and provides a HTTP API
- * a [[converter]] that encapsulates [Metafacture](https://github.com/culturegraph/metafacture-core) to transform data
+ * a [[converter]] that encapsulates [metafacture](https://github.com/culturegraph/metafacture-core) to transform data
  * and a [[persistence]] layer to access the [[metadata repository|Glossary#metadata-repository]] (currently a relational database; MySQL) and the [[data hub|Glossary#data-hub]] (currently a graph database; [Neo4j](http://www.neo4j.org)).
- 
-End users, e.g., system librarians, usually interact with the [Backoffice we application](http://demo.dswarm.org) and may use the [[data hub browser|Graph-Exploration]] to inspect or visualize the data stored by d:swarm.
 
-[![an image of the architecture overview](https://raw.githubusercontent.com/wiki/dswarm/dswarm-documentation/img/architecture.png)](https://raw.githubusercontent.com/wiki/dswarm/dswarm-documentation/img/architecture.png "Architecture Overview")
+[[img/architecture.png]]
 
+Users, e.g., system librarians, usually interact with the [BackOffice web application](http://demo.dswarm.org). Just like the TPU, which batch-processes ingest, transformation and export tasks, it communicates with the d:swarm backend via the controller's HTTP API.
 The HTTP API provides a documentation via [[Swagger|http://swagger.io/]] and hence can be explored via the [[Swagger UI|HTTP API Discovery]]. This is a very convenient way to explore the back end's functionality. 
 
 ## How to get started?

```

Comment:
do still need this paragraph at this position here? - if this is the case, then we should link to the feedback paragraph as well from here